far/FarEng.hlf.m4

```diff
@@ -2555,15 +2558,15 @@ $ #Settings dialog: panel#
 
 @TreeSettings
 $ #Tree settings#
-  #Auto change folder#      If checked, cursor moves in the ~tree panel~@TreePanel@
-                          will cause a folder change in the other
-                          panel. If it is not checked, you must press
-                          #Enter# to change the folder from the tree
-                          panel.
+  #Auto change folder#      If turned on, moving cursor on the ~tree panel~@TreePanel@
+                          will synchronously change the folder on the other panel.
+                          If turned off, to change the folder from the tree panel,
+                          you need to press #Enter#.
+                          Related ~far:config~@FarConfig@ parameter is #Panel.Tree.AutoChangeFolder#.
 
-  #Минимальное#             Минимальное количество папок на диске,
-  #количество папок#        после которого будет создаваться файл
-                          #tree3.far#.
+  #Minimum number#          The minimal number of folders on the disk for which
+  #of folders#              folder tree cache file #tree3.far# will be created.
+                          Related ~far:config~@FarConfig@ parameter is #Panel.Tree.MinTreeCount#.
```

Comment:
No need to specify far:config equivalent for every option,

Reply:
Removed.